main/src/library/Indexable.flix

```diff
@@ -41,3 +41,11 @@ pub trait Indexable[t] {
     pub def get(i: Indexable.Idx[t], t: t): Indexable.Elm[t] \ Indexable.Aef[t]
 
 }
+
+mod Indexable {
+
+    /// Alias of `get` with arguments in syntactic order.
+    pub def getSwapped(t: t, i: Indexable.Idx[t]): Indexable.Elm[t] \ Indexable.Aef[t] with Indexable[t] =
```

Comment:
I assume this is to preserve evaluation order.

But what about swapping the arguments to the get/put signatures?

Reply:
we could also do that yes, just wanted to get it working. i don't think i have strong opinions on which solution